### PR TITLE
databases-week2

### PR DESCRIPTION
**Pull request by negaren**

### CODE REVIEW

**Review by mani-zaeim — COMMENTED** [review]
the Director_ID is wrongly added to Movie table while you have created conjunction table as movie_actor, please remove it from Movie

**Comment by mani-zaeim** [comment]
Good job, Please correct the code and re commit

File: databases/week2/database_week2_homework.sql [hunk 1]
```diff
@@ -28,29 +25,29 @@ UPDATE
 set
   title = 'Planting'
 WHERE
-  user_id = 12;
+ id = (select LAST_INSERT_ID());
```
Comment (mani-zaeim) [inline databases/week2/database_week2_homework.sql:28]: nice, very interesting use of sub select and default methods

File: databases/week2/school.sql [hunk 2]
```diff
@@ -13,20 +14,23 @@ CREATE TABLE `student` (
...
 ALTER TABLE
   `class`
 ADD
-  CONSTRAINT chk_status Check(
-    `status` = 'not-started'
-    OR `status` = 'ongoing'
-    OR `status` = 'finished'
-  );
\ No newline at end of file
+  `status` ENUM('not-started', 'ongoing', 'finished');
```
Comment (mani-zaeim) [inline databases/week2/school.sql:31]: Good job

**Review by mani-zaeim — COMMENTED** [review]
Very nice job Negar, well done

File: databases/week2/movie.sql [hunk 3]
```diff
@@ -0,0 +1,32 @@
+USE movie;
+
+CREATE TABLE `director`(
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+    PRIMARY KEY(`id`),
```
Comment (martamatos) [inline databases/week2/movie.sql:5]: as good practice, either define the PRIMARY KEY like this after defining all attributes, or define the PRIMARY KEY in the same line as the  `id` attribute.

**Pull request merged**